docs/user/faq.rst

```diff
@@ -446,9 +446,14 @@ for this exception type. Alternatively, you may be able to configure your web
 server to transform the response for you (e.g., using Nginx's ``error_page``
 directive).
 
-500 errors are typically the result of an unhandled exception making its way
-up to the web server. To handle these errors more gracefully, you can add a
-custom error handler for Python's base :class:`Exception` type.
+By default, Falcon handles all exceptions not inheriting from
```

Comment:
Technically, _all exceptions_ is not 100% correct since we are not installing any handler for `BaseException` (neither should we). But probably we don't need to dig that deep in the FAQ, this will do :+1:

Reply:
Fair point. I have to fix the commit message style anyway, so I can go make this technically correct

Reply:
Oh, sorry for stepping on your toes... I have hopefully addressed Sphinx complaints via the Web UI

Reply:
No worries :-)

Reply:
And btw don't worry about the commit message style. We maintainers get an extra chance to edit that when squash-merging a pull request. (But we seldom get it right ourselves...)

Reply:
Okay, I'm just double-checking that the docs are formatted how I wanted, and then I'll push and we should be good to go